tensorflow_model_optimization/python/core/common/keras/compression/internal/optimize.py

```diff
@@ -112,15 +112,15 @@ def build(self, input_shape):
 
   def call(self, inputs):
     for attr_name in self.compressible_weights:
-      training_weight_tensors = [
-          v.read_value() for v in self.training_weights[attr_name]
-      ]
-      weight_tensor = self.algorithm.training(training_weight_tensors)
       # TODO(tfmot): move this to the inference graph only, since
```

Comment:
Update comment to "move constant folding prevention to the inference graph only ..."